addons/resource/__openerp__.py

```diff
@@ -18,13 +18,7 @@
     'data': [
         'security/ir.model.access.csv',
         'security/resource_security.xml',
-        'resource_view.xml',
+        'views/resource_view.xml',
     ],
-    'demo': ['resource_demo.xml'],
-    'test': [
-        'test/resource.yml',
-        'test/duplicate_resource.yml',
-    ],
-    'installable': True,
-    'auto_install': False,
+    'demo': ['demo/resource_demo.xml'],
```

Comment:
In the same file (demo/resource_demo.xml), remove `<openerp>` and `<data>` tags, use `<odoo>` instead.